scripts/v1alpha1/create_machine.sh

```diff
@@ -9,10 +10,17 @@ V1ALPHA1_SCRIPTS_PATH="$(dirname "$(readlink -f "${0}")")/"
 USER_DATA_SCRIPT_NAME="user_data.sh"
 SCRIPT_PATH="${V1ALPHA1_SCRIPTS_PATH}""${USER_DATA_SCRIPT_NAME}"
 
+if [[ "${PROVISIONING_IPV6}" == "true" ]]; then
+  PROVISIONING_HOST="[fd2e:6f44:5dd8:b856::1]"
+else
+  PROVISIONING_HOST=172.22.0.1
+fi
```

Comment:
Can we just use `PROVISIONING_IP` from `lib/common.sh` here?

Reply:
Ah yes, thanks, done